bazel/README.md

```diff
@@ -215,6 +215,11 @@ By default Clang drops some debug symbols that are required for pretty printing
 More information can be found [here](https://bugs.llvm.org/show_bug.cgi?id=24202). The easy solution
 is to set ```--copt=-fno-limit-debug-info``` on the CLI or in your .bazelrc file.
 
+## Removing debug info
+
+If you don't want your debug or release binaries to contain debug info
+to reduce binary size, pass `--define=no_debug_info=1` when building.
```

Comment:
This remind me a bit, do you want to add this into .bazelrc for config=sizeopt?

Reply:
I don't particularly think so because folks could use the `.stripped` target or `--strip=always` in the "normal" case of this, so opting them into this when they just want "normal" size reductions might be a bit too aggressive.

Reply:
I think it worth a comment here mentioning `stripped` and `--config=sizeopt` as an alternative option?

Reply:
Good idea, added those alternatives and mentioned this is useful for static libs

Reply:
Thanks!